packages/appeals-service-api/src/spec/api-types.d.ts

```diff
@@ -177,6 +177,17 @@ export interface AppealSubmission {
 		lpaCode?: string;
 		/** @format date-time */
 		decisionDate?: string;
+		appealType?: string;
```

Comment:
do these get generated from the openapi definition or are we handcrafting them?

Reply:
Generated, see: https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/main/docs/api-spec.md#updates